src/Elastic.Markdown/Myst/FrontMatter/ApplicableTo.cs

```diff
@@ -78,11 +78,15 @@ public record DeploymentApplicability
 	[YamlMember(Alias = "ess")]
 	public AppliesCollection? Ess { get; set; }
 
+	[YamlMember(Alias = "ech")]
+	public AppliesCollection? Ech { get; set; } // ess alias, preferred
+
 	public static DeploymentApplicability All { get; } = new()
 	{
 		Ece = AppliesCollection.GenerallyAvailable,
 		Eck = AppliesCollection.GenerallyAvailable,
 		Ess = AppliesCollection.GenerallyAvailable,
+		Ech = AppliesCollection.GenerallyAvailable,
```

Comment:
I don't think this should be a new property, rename `Ess` to `Ech` and have both `ess` and `ech` assign to `Ech`. 

Specifying both should be an error.